src/lib/containers/download_list_v2/DirectProgrammaticDownload.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
```

Comment:
Similar to DirectDownload, but this takes care of the button (code icon) and modal (uses ProgrammaticInstructionsModal)